### DISCUSSION TITLE
Smart Connections 2.1 is breaking Mobile Obsidian App

### DISCUSSION THREAD

**Discussion by Over-The-Edge** [brianpetro/obsidian-smart-connections#518] [Q&A]
I am running latest Smart Connections 2.1 version in Obsidian and works great on Laptop, but when enabled causes Obsidian Mobile to keep looping on startup and won't let me access my notes, even if I press "skip". If I disable Smart Connections Plugin on Desktop then I can login on Mobile. Suspect something is causing plugin to run on Mobile when it was previously not able to run on Mobile App.

**Comment by brianpetro** [comment]
@Over-The-Edge This has been fixed in `v2.1.14`. The Smart Connections early release version should automatically update the next time you restart Obsidian/Smart Connections.

For background on the issue, I've been working on Smart Connections mobile compatibility, and since the early release version is essentially my development branch, that's how that got left in there. 

Sorry for the trouble!

🌴

**Reply by brianpetro** [reply]
@Over-The-Edge and thanks for bringing this to my attention!

**Reply by Over-The-Edge** [reply]
@brianpetro thanks for the quick fix all working now, and excited to hear that you are working on a mobile version!
